packages/ringcentral-integration/modules/CallMonitor/index.js

```diff
@@ -260,6 +261,25 @@ export default class CallMonitor extends RcModule {
         });
       },
     );
+    this.addSelector('ringoutRingCalls',
+      this._selectors.calls,
```

Comment:
Why not add an selector which will get all the ringout calls, and then compose other selectors by using the selector. Other than filtering upon the original `this._selectors.calls`